packages/flutter/lib/src/material/material.dart

```diff
@@ -200,15 +201,17 @@ class Material extends StatefulWidget {
   /// {@template flutter.material.material.elevation}
   /// The z-coordinate at which to place this material relative to its parent.
   ///
-  /// This controls the size of the shadow below the material.
+  /// This controls the size of the shadow below the material and the dark
+  /// theme overlay color if inside a dark theme.
   ///
   /// If this is non-zero, the contents of the material are clipped, because the
   /// widget conceptually defines an independent printed piece of material.
   ///
-  /// Defaults to 0. Changing this value will cause the shadow to animate over
-  /// [animationDuration].
+  /// Defaults to 0. Changing this value will cause the shadow and dark theme
+  /// overlay to animate over [animationDuration].
```

Comment:
and dark => and the

I wish there was a link to an explanation of "dark theme overlay" in here somewhere. Maybe insert a `[applyDarkThemeOverlay]` somewhere.